CMP1903_A1_2324/Testing.cs

```diff
@@ -16,5 +17,31 @@ internal class Testing
          */
 
         //Method
+        public void Test()
+        {
+            // Runs a test versio of the game, and checks the outputs of the rolls
+            Game testGame = new Game();
+            testGame.ExecuteGame();
+
+            if (testGame.TotalOfRolls() < 18)
+            {
+                if (testGame.TotalOfRolls() > 3)
+                {
+                    Console.WriteLine("Total Is As Expected: " + testGame.TotalOfRolls());
+                }
+                else
+                {
+                    Console.WriteLine("Total Was Less Than 3, Which Is Impossible: " + testGame.TotalOfRolls());
+                }
+            }
+            else
+            {
+                Console.WriteLine("Total Was Greater Than 18, Which Is Impossible: " + testGame.TotalOfRolls());
+            }
+            
+
+        }
+
+
     }
 }
```

Comment:
Commenting is non-existent here, however isn't as necessary, although still would have been nice to include some. Didn't use Debug.Assert(), however error handling is seen here.

Reply:
Commenting is pretty much dont for you by the console.writelines as the output explains the statement they are in. As for the Debug.Assert() i have now added that in, thank you for catching that for me!